### ISSUE TITLE
Cross Compiling 

### ISSUE THREAD

**Issue by ursus-the-bear** [ossec/ossec-hids#1113]
I would like to cross compile for the Raspberry Pi 3 on Ubuntu 17.04 but cannot seem to find any information on this process. I can find cross compiling for another server but not for another architecture. I "just" need to compile the agent which I will be adapting later and therefore do not want to compile (and later adapt) on the Pi.  Any pointers?

**Comment by ddpbsd** [comment]
How do you cross compile other programs?

**Comment by ursus-the-bear** [comment]
I had to do something similar in op-tee (https://github.com/OP-TEE/build) and they have build files that allow me to add the platform when I download the code from git. It is then very easy to do the cross compile. I was wondering if there was something similar for ossec or if I needed to setup the environment myself (something I have never done).

I have found this: http://stackoverflow.com/questions/19162072/installing-raspberry-pi-cross-compiler/19269715#19269715 but wanted to save myself the work incase I just missed the setup for ossec.
